firestore/google/cloud/firestore_v1/transaction.py

```diff
@@ -200,6 +202,37 @@ def _commit(self):
         self._clean_up()
         return list(commit_response.write_results)
 
+    def get_all(self, references):
+        """Retrieves multiple documents from Firestore.
+
+        Args:
+            references (List[.DocumentReference, ...]): Iterable of document
+                references to be retrieved.
+
+        Yields:
+            .DocumentSnapshot: The next document snapshot that fulfills the
+            query, or :data:`None` if the document does not exist.
+        """
+        return self._client.get_all(references, transaction=self._id)
+
+    def get(self, ref_or_query):
+        """
+        Retrieve a document or a query result from the database.
+        Args:
+            ref_or_query The document references or query object to return.
+        Yields:
+            .DocumentSnapshot: The next document snapshot that fulfills the
+            query, or :data:`None` if the document does not exist.
+        """
+        if isinstance(ref_or_query, DocumentReference):
+            return self._client.get_all([ref_or_query], transaction=self._id)
+        elif isinstance(ref_or_query, Query):
+            return ref_or_query.stream(self._id)
+        else:
+            raise ValueError(
+                'Value for argument "refOrQuery" must be a DocumentReference or a Query.'
+            )
```

Comment:
This should better be `Value for argument "ref_or_query"...`

Reply:
I think, we should raise a TypeError, not ValueError here:
raise TypeError('ref_or_query" must be of type  DocumentReference or a Query.')

Reply:
ok i will do that